components/Button/src/index.scss

```diff
@@ -165,3 +165,18 @@ a.denhaag-button {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
+.denhaag-button--icon-only {
+  --denhaag-button-medium-size-padding-block: 0.6875rem;
+  --denhaag-button-medium-size-padding-inline: 0.75rem;
+
+  padding-block-start: var(--denhaag-button-medium-size-padding-block);
```

Comment:
This paddings are set on the default `denhaag-button` selector on line #13 til #16. No need for duplicate CSS